include/solvers/navier_stokes_scratch_data.h

```diff
@@ -908,6 +908,7 @@ class NavierStokesScratchData
   std::map<field, std::vector<double>> fields;
   std::vector<double>                  density;
   std::vector<double>                  viscosity;
+  double                               viscosity_scale;
```

Comment:
If this is only used in the Nitsche assembler, it should only be given to the Nitsche assembler from my POV.